x-pack/plugins/enterprise_search/public/applications/app_search/components/api_logs/api_logs.tsx

```diff
@@ -47,19 +56,27 @@ export const ApiLogs: React.FC<Props> = ({ engineBreadcrumb }) => {
       <FlashMessages />
       <LogRetentionCallout type={LogRetentionOptions.API} />
 
-      <EuiFlexGroup gutterSize="m" alignItems="center" responsive={false} wrap>
-        <EuiFlexItem grow={false}>
-          <EuiTitle size="s">
-            <h2>{RECENT_API_EVENTS}</h2>
-          </EuiTitle>
-        </EuiFlexItem>
-        <EuiFlexItem grow={false}>
-          <LogRetentionTooltip type={LogRetentionOptions.API} />
-        </EuiFlexItem>
-        <EuiFlexItem grow={false}>{/* TODO: NewApiEventsPrompt */}</EuiFlexItem>
-      </EuiFlexGroup>
+      <EuiPageContent hasBorder>
+        <EuiPageContentBody>
+          <EuiFlexGroup gutterSize="m" alignItems="center" responsive={false} wrap>
```

Comment:
Everything inside here is [mostly] the same as before, I just added `EuipageContent` and `EuiPageContentBody` wrappers - I recommend viewing with hidden whitespace changes for easier diffing

Reply:
Why do I see this comment here on the Conversation tab, but not on the Files changed tab? Maybe because of a rebase or something.